eiffel-vocabulary/EiffelArtifactDeployedEvent.md

```diff
@@ -0,0 +1,19 @@
+
+# EiffelArtifactDeployedEvent (ArtD)
+
+The EiffelArtifactDeployedEvent declares that a software artifact (declared by EiffelArtifactCreatedEvent) has been deployed into an Environment (declared by EiffelEnvironmentDefinedEvent) and is consequently available for use.
```

Comment:
I'm not sure I get the "available for use" part. Wouldn't that more accurately reflect EiffelServiceStartedEvent? I would expect something like "and can be launched (typically represented by EiffelServiceStartedEvent).", where EiffelServiceStartedEvent is a link.